istioctl/pkg/dashboard/dashboard.go

```diff
@@ -604,3 +567,16 @@ func Dashboard(cliContext cli.Context) *cobra.Command {
 
 	return dashboardCmd
 }
+
+func inferPodMeta(ctx cli.Context, client kube.CLIClient, labelSelector string) (name, namespace string, err error) {
+	for _, ns := range []string{ctx.IstioNamespace(), ctx.NamespaceOrDefault(ctx.Namespace())} {
+		pl, err := client.PodsForSelector(context.TODO(), ns, labelSelector)
+		if err != nil {
+			return "", "", fmt.Errorf("not able to locate pod with selector %s: %v", labelSelector, err)
```

Comment:
How does this error work with empty items check below?  Is this `err` just a connection/permission error on the request?  Or does it also return error if it finds no pods?  Either way, we should probably skip this error check and rely on the default empty return on ln 581 since there may be a pod returned by the second namespace lookup

Reply:
> Is this err just a connection/permission error on the request?

Should be this kind of errors.

> Either way, we should probably skip this error check and rely on the default empty return on line 581 since there may be a pod returned by the second namespace lookup

Makes sense, thanks for the suggestion!